### ISSUE TITLE
id selector does not seem to work

### ISSUE THREAD

**Issue by clouddra** [airbnb/enzyme#64]
Was trying to test a simple piece of code:
```javascript
const renderShallow = shallow(<div id='ttt' className='ttt'>hello</div>);
console.log(renderShallow.find('#ttt')); // not found
console.log(renderShallow.find('.ttt')); // found
```
and it returns an empty array when i tried to use an id selector. It works fine on class selectors.

**Comment by clouddra** [comment]
Just to add, I had the following configuration in my karma config. Not sure if it affects for shallow rendering.
```
externals: {
        'jsdom': 'window',
        'cheerio': 'window',
        'react/lib/ExecutionEnvironment': true,
}
```

**Comment by lelandrichardson** [comment]
Hi @clouddra,  can you confirm that the following code works?

```jsx
const wrapper = shallow(
  <div>
    <div id='ttt' className='ttt'>hello</div>
  </div>
);
console.log(wrapper.find('#ttt').length); // 1
```

There may be some edge-case behavior when the id you are looking for is on the root node... especially if you're using React 0.14, where shallow rendering of a DOM Element as opposed to a composite component is behavior shimmed by enzyme rather than built in to the shallow renderer.

**Comment by lelandrichardson** [comment]
Confirmed the issue and fixed. Will publish to npm later today when this lands.

**Comment by lelandrichardson** [comment]
@clouddra  The fix is now available in v1.2.0

**Comment by clouddra** [comment]
@lelandrichardson That was fast! Tested it and it works fine. Thanks a lot for your help.
